bullet_train-themes-light/app/views/showcase/previews/partials/_link_row.html.erb

```diff
@@ -0,0 +1,11 @@
+<% showcase.sample "Basic" do %>
+  <%= render "shared/link_row", path: new_user_session_path do |link_row| %>
+    <% link_row.body "Login Page" %>
+  <% end %>
+<% end %>
+
```

Comment:
We could do a `sample "Without body"`?

It would be nice with one that has more realistic content in `body`, but I can't find real renders of `link_row`. We can find that another time.